open-company/open-company-web
# Video image instead of player to improve performance.

Load an image and our own spinner instead of the ziggeo player so that mobile devices perform better.

Needs https://github.com/open-company/open-company-storage/pull/155

To test:

- create a few video posts
- and reload all posts with a few of the videos displayed.
- [x] do you see the video image with a play button?
- [x] is it an image and not the ziggeo player (inspect the html for the img tag)
- [x] can you click the image to play the video?

- create a video post
- [x] does the ziggeo player load after posting instead of displaying the img?

- open a display of all posts with a second user
- create a video post as the first user
- [ ] as the second user do you see the new post with a spinner?
